README.md

```diff
@@ -14,7 +14,15 @@
 [![Test Coverage](https://codeclimate.com/github/solnic/transproc/badges/coverage.svg)][codeclimate]
 [![Inline docs](http://inch-ci.org/github/solnic/transproc.svg?branch=master)][inchpages]
 
-Transproc is a small library that allows you to compose methods into a functional pipeline using left-to-right function composition. It works like `|>` in Elixir or `>>` in F#.
+Transproc is a small library that allows you to compose methods into a functional
+pipeline using left-to-right function composition.
+
+The approach came from Functional Programming, where small functions are composed
```

Comment:
Not sure about `small functions are composed into bigger functions`, how about `simple functions are composed into more complex functions` or something along those lines?